src/components/UserCard/index.jsx

```diff
@@ -26,17 +35,61 @@ const useStyles = makeStyles(theme => ({
   cardHeaderAction: {
     alignSelf: 'end',
   },
+  cardActions: {
+    justifyContent: 'flex-end',
+  },
   pencilIcon: {
     marginRight: theme.spacing(1),
   },
   link: {
     ...theme.mixins.link,
   },
+  scheduledChangeDescriptionMarginLeft: {
+    lineHeight: '24px',
+    marginLeft: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+  },
+  scheduledChangeDescription: {
+    lineHeight: '24px',
+    marginBottom: theme.spacing(2),
+  },
```

Comment:
Instead of having `marginBottom` on both `scheduledChangeDescriptionMarginLeft` and `scheduledChangeDescription`, we could define it once on the Grid container.